CMakeLists.txt

```diff
@@ -34,3 +43,13 @@ install(DIRECTORY include/class_loader/
 if(CATKIN_ENABLE_TESTING)
   add_subdirectory(test)
 endif()
+
+set(TARGET_NAME ${PROJECT_NAME})
+set(PKGCONFIG_LIBS ${Boost_LIBRARIES} ${console_bridge_LIBRARIES} ${Poco_LIBRARIES})
+
+# Prepare and install necessary files to support finding of the library 
+# using pkg-config
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/src/${TARGET_NAME}.pc.in
+            ${CMAKE_CURRENT_BINARY_DIR}/${TARGET_NAME}.pc @ONLY)
+
+install(FILES ${CMAKE_BINARY_DIR}/${PROJECT_NAME}.pc DESTINATION lib/pkgconfig)
```

Comment:
This should not be done if `catkin` is available since it generates its own pkgconfig file.